pkgs/development/python-modules/pymupdf/default.nix

```diff
@@ -76,7 +77,7 @@ buildPythonPackage rec {
     gumbo
   ] ++ lib.optionals (stdenv.system == "x86_64-darwin") [ memstreamHook ];
 
-  propagatedBuildInputs = [ mupdf-cxx ];
+  propagatedBuildInputs = [ (toPythonModule mupdf-cxx) ];
```

Comment:
In [`4a3db5a` (#298783)](https://github.com/NixOS/nixpkgs/pull/298783/commits/4a3db5a7999aff6fb1dfcd9c33a5cfb64dd8770e) @dotlambda proposed to move it to buildInputs. What is more correct? I am undecided.

Reply:
Are buildInputs even included in the environment when using pythonWithPackages envs? Mupdf itself provides a python module, so it needs to be part of the env, and I don't see how moving this to buildInputs changes that?

Reply:
But I am not 100% familiar with the python nix packaging. From the documentation, I think the most correct would be to put it in "dependencies"?

Reply:
Yes, `dependencies` would be best. However it is currently roughly equivalent to `propagatedBuildInputs` anyway.

Reply:
`dependencies` sounds correct to me.